configuration/968b0/delegate-keys.000.key

```diff
@@ -0,0 +1 @@
+X��nPϕF��W߄.f"�"���j����;6OZ���o��H*��poI�FP�}J�g��>��N�	��'�{c|��ߞ(rwA�𵆐�iH�\Cb^�\lǣ��P��XV�]ѤDãg^e
```

Comment:
> call script: bash scripts/genesis.sh to create genesis file, keys and certificatesnew genesis for demo

Can we automate this part either as a separate script or part of your current script (whichever makes more sense) so we don't have additional files cluttering `cardano-node`?

Reply:
this is Serge's script. Its output is in a directory called `genesis.1566487374` (the last part being a timestamp). For this step to be automated, we need to choose the name of the output directory. Or, it combines steps 2+3+4 and puts everything in configuration/ based on the hash; and, updates the scripts. Will retry tomorrow again..

Reply:
the script now outputs the created files in a directory under `configuration/`

Reply:
@CodiePP Great we can remove the `968b0` dir now?

Reply:
After talking to @CodiePP, we decided to leave the `968b0` to avoid having to generate the files each time before running the demo.